src/Shellbar/Shellbar.js

```diff
@@ -479,8 +482,8 @@ Shellbar.propDescriptions = {
     },
     notifications: 'Information about pending notifications.',
     productMenu: 'Holds product titles and navigation.',
-    productSwitcher: 'For navigating between products.',
-    productSwitcherList: 'List of the products.',
+    productSwitch: 'For navigating between products.',
+    productSwitchList: 'List of the products.',
```

Comment:
Should this be more descriptive - that it's an array of objects? Name the key options might be a good idea and that title and callback are required with subtitle being optional.